python/grep.py

```diff
@@ -417,8 +420,7 @@ def get_config_log_filter():
 
 def get_home():
     home = weechat.config_string(weechat.config_get('logger.file.path'))
-    home = path.abspath(path.expanduser(home))
-    return home.replace('%h', weechat.info_get('weechat_dir', ''))
+    return weechat.string_eval_path_home(home, '', '', '')
```

Comment:
I started a review on your old code, now the string_eval_path_home has gone with your latest changes, so you can just ignore my comment.